packages/block-library/src/pattern/block.json

```
@@ -0,0 +1,17 @@
+{
+	"apiVersion": 2,
+	"name": "core/pattern",
+	"title": "pattern",
+	"category": "text",
```

Possibly `design` or `theme`?

```suggestion
	"category": "design",
```